pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.532</version>
+    <version>1.610</version>
```

Comment:
Jenkins has a bug https://issues.jenkins-ci.org/browse/JENKINS-18537 that has problems with JDK8.